cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTable.java

```diff
@@ -54,9 +55,9 @@
   private final HTable hTable;
 
   public HBaseMetricsTable(String name, Configuration hConf) throws IOException {
-    String hTableName = HBaseTableUtil.getHBaseTableName(name);
+    String hTableName = HTableNameConverter.getHBaseTableName(name);
     HBaseTableUtil tableUtil = new HBaseTableUtilFactory().get();
-    HTable hTable = tableUtil.getHTable(hConf, TableId.from(hTableName));
+    HTable hTable = tableUtil.createHTable(hConf, TableId.from(hTableName));
```

Comment:
Yes, this will be fixed when we have new ``DatasetDefinition`` APIs available. Coming in another PR.